src/components/interactive/interactive.component.tsx

```diff
@@ -52,7 +53,13 @@ export const Interactive = memo(({ onCoordinateChange, children }: IInteractiveP
   );
 
   return (
-    <div ref={interactiveRef} className="rcp-interactive" onMouseDown={onStart} onTouchStart={onStart}>
+    <div
+      ref={interactiveRef}
+      className="rcp-interactive"
+      onMouseDown={onStart}
+      onTouchStart={onStart}
+      aria-readonly={disabled}
```

Comment:
For the `Interactive` component, it would be more appropriate to use `aria-disabled` instead of `aria-readonly`, since there's nothing to read from it and all the useful information is in the input fields that have the `readonly` attribute.